src/components/GridDisplay.jsx

```diff
@@ -2,19 +2,19 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import PostGrid from './PostGrid.jsx';
 
-const GridDisplay = ({ fetchedData }) => (
+const GridDisplay = ({ fetchedData, user }) => (
```

Comment:
Now each component that needs authenticated access will need a user prop added. For now this is fetched on demand or received through props. This works for now but can be added to either a cookie or context api.